modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/utils/DeployEnvironments.java

```diff
@@ -99,7 +97,7 @@ public static Map<String, String> getVariables(DeployTask task) {
                 variables.put(variable.getName(), variable.getValue());
             }
         }
-
+        variables.putAll(getFlavorVariables(task));

```

Comment:
I suggest we leave the kinds of variables to be assembled in the terraform deployment. 
I am wondering if all deployment implementations will do it the same way.

Reply:
@iskey ok understood. I will move this to Terraform specific classes.